source/javascripts/gradient_fade.js

```diff
@@ -0,0 +1,16 @@
+function init() {
+    // Create the listener function
+    var updateLayout = _.debounce(function(e) {
```

Comment:
Page scroll is feeling a little choppy from my end (though my computer is pretty old at this point). I'm wondering if there is some CSS-only way of doing this. It's really just positioning a gradient at the top of the page right? Is there any reason it needs to update on scroll? Could you just make a gradient once and be done with it?